### PR TITLE
Setup blunderbuss and tide for Kubeflow

### PR DESCRIPTION
**Pull request by jlewi**
*  We want to use blunderbuss to start assigning reviewers
* We want to use tide to start automerging approved PRs.

Related to kubeflow/community#19
Related to kubeflow/community#20

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -41,6 +41,21 @@ branch-protection:
 
 tide:
   queries:
+  - repos:
+    - kubeflow/community
```
Comment (cjwagner) [inline prow/config.yaml]: This repo does not have a root level OWNERS file yet which will be needed in order for the approval process to work.

**Comment by krzyzacy** [comment]
hummm, run config sort?

also, does kubeflow as OWNERS file setup?

File: prow/config.yaml [hunk 2]
```diff
@@ -102,6 +117,11 @@ tide:
     - do-not-merge/hold
     - do-not-merge/work-in-progress
   merge_method:
+    kubeflow/community: squash
+    kubeflow/examples: squash
+    kubeflow/kubeflow: squash
+    kubeflow/testing: squash    
+    kubeflow/tf-operator: squash
```
Comment (cjwagner) [inline prow/config.yaml]: `kubeflow: squash` is sufficient here.

File: prow/config.yaml [hunk 3]
```diff
@@ -102,6 +117,11 @@ tide:
     - do-not-merge/hold
     - do-not-merge/work-in-progress
   merge_method:
+    kubeflow/community: squash
+    kubeflow/examples: squash
+    kubeflow/kubeflow: squash
+    kubeflow/testing: squash
+    kubeflow/tf-operator: squash
```
Comment (cjwagner) [inline prow/config.yaml]: `kubeflow: squash` is sufficient here.

**Comment by jlewi** [comment]
@krzyzacy Yup have (or will add OWNERs files) for all these repositories.

File: prow/config.yaml (hunk 1 above)
Comment (jlewi) [reply prow/config.yaml]: Thanks. We'll add one. Any reason I need to do that before this PR?

Comment (cjwagner) [reply prow/config.yaml]: Shouldn't matter much either way, but nothing will be able to get the approved label in that repo until the there is an OWNERS file so you'd have to click the merge button to merge the PR to add the OWNERS file.

Comment (jlewi) [reply prow/config.yaml]: Ack

**Comment by jlewi** [comment]
@cjwagner Done.

If the bot can squash commits why don't you just let the bot squash the commits? Is it just to get a good commit message?

**Comment by cjwagner** [comment]
> If the bot can squash commits why don't you just let the bot squash the commits? Is it just to get a good commit message?

Squashing an entire PR into a single commit doesn't always make sense. For this PR it does, but for larger PRs with multiple parts it is helpful to break up the changes into separate commits to make reviewing and cherrypicking easier and more granular.

**Pull request merged**